manager/migrations/0032_db_collation.py

```diff
@@ -0,0 +1,43 @@
+from django.db import migrations
+
+
+def alter_collation(character_set, collation, schema_editor):
+	with schema_editor.connection.cursor() as cursor:
+		# set for database
+		print('Altering database…')
+		cursor.execute(
+			f'ALTER DATABASE CHARACTER SET {character_set} COLLATE {collation};'
+		)
+		# set for tables (and convert data)
+		cursor.execute(
+			'SHOW TABLES;'
+		)
+		for table in cursor.fetchall():
+			print(f'Altering table `{table}`…')
+			cursor.execute(
+				f'ALTER TABLE {table} CONVERT TO CHARACTER '
```

Comment:
Documentation on this, for those interested:

https://dev.mysql.com/doc/refman/8.0/en/alter-table.html#alter-table-character-set